source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -839,9 +851,11 @@ namespace aspect
                                    "two opposing points (SW and NE or NW and SE) a coordinate parallel ellipsoidal "
                                    "chunk geometry will be created. 2) by defining three points a non-coordinate "
                                    "parallel ellipsoidal chunk will be created. The points are defined in the input "
-                                   "file by longitude:latitude. It is also possible to define additional subdivisions of the "
-                                   "mesh in each direction. Faces of the model are defined as 0, west; 1,east; 2, south; 3, "
-                                   "north; 4, inner; 5, outer.\n\n"
+                                   "file by longitude:latitude. It is also possible to define additional subdivisions "
+                                   "of the mesh in each direction. The boundary of the domain joining two adjacent points "
+                                   "satisfies the expression [lon, lat](f) = [lon1*f + lon2*(1-f), lat1*f + lat2*(1-f)], "
+                                   "where f is a value between 0 and 1. Faces of the model are defined as "
```

Comment:
To me this looks like a complicated way to say that there is a linear interpolation between the coordinates. Is that correct or am I misinterpreting it?

Reply:
That's correct - I'll add a few more words to that effect, but will keep the math in unless you have strong objections.